src/AvField.js

```diff
@@ -89,14 +89,14 @@ export default class AvField extends Component {
       size={size}
       disabled={disabled}
       readOnly={readOnly}
-      {...attributes}
+      {...{...attributes, ...{'aria-describedby':'errorMsg'}}}
     >
       {children}
     </AvInput>);
 
     const validation = this.context.FormCtrl.getInputState(this.props.name);
 
-    const feedback = validation.errorMessage ? (<AvFeedback>{validation.errorMessage}</AvFeedback>) : null;
+    const feedback = validation.errorMessage ? (<AvFeedback id="errorMsg">{validation.errorMessage}</AvFeedback>) : null;
```

Comment:
We should make the id dynamic. If multiple fields have an error, then there will be more than one element with `id="errorMsg"`.

We can generate it off the id for the field
```
<AvFeedback id={`${id}-error`} ...`
```

we should use this value with the `aria-describedby` prop as well